pubspec.yaml

```diff
@@ -30,18 +32,28 @@ dependencies:
   intro_slider:
     git:
       url: https://github.com/shareef-dweikat/flutter-intro-slider
+  window_size:
+    git:
+      url: git://github.com/google/flutter-desktop-embedding.git
+      path: plugins/window_size
   flutter:
     sdk: flutter
 
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
   app_settings: ^4.1.1
+  desktop_window: ^0.4.0
```

Comment:
@ewanas, do you know what this is for?

Reply:
No idea,  @shareef-dweikat any reason we needed this dependency?

Reply:
@ewanas @bryanchriswhite  I used this dependency to resize mobile UI to fit desktop. We don't need it now because we have totally separate UI for desktop.